global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.1.23115.2",
+    "version": "8.0.100-preview.7.23376.3",
     "allowPrerelease": false,
-    "rollForward": "latestPatch"
+    "rollForward": "disable"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.1.23115.2",
+    "dotnet": "8.0.100-preview.7.23376.3",
     "vs": {
-      "version": "17.4.1"
+      "version": "17.8.0"
```

Comment:
This branch should never be newer than 17.6 (we should build and test in the same version it services).

Reply:
It is great is theory, however, we don't control our CI images and they are updated regularly with newer versions VS. I will downgrade this back to 17.6, although, I am unsure of what effect that will have when running on an image with 17.10 installed.